### PR TITLE
Improve front end tests

### PR DESCRIPTION
**Pull request by Ugzuzg**
## Issue

https://github.com/electricitymaps/electricitymaps-contrib/issues/4550

## Description

- Upgrade Cypress to 10
- Make sure WebGL is enabled in Chromium browsers

### CODE REVIEW

**Review by VIKTORVAV99 — APPROVED** [review]
Everything looks good as far as I can tell but I'll wait for @tonypls to do a review as well before merging.

Thanks for taking the time to improve our testing!

File: web/cypress/plugins/index.js [hunk 1]
```diff
@@ -19,4 +19,12 @@
 module.exports = (on, config) => {
   // `on` is used to hook into various events Cypress emits
   // `config` is the resolved Cypress config
+  on('before:browser:launch', (browser = {}, launchOptions) => {
+    if (browser.family === 'chromium' && browser.name !== 'electron') {
+      return {
+        ...launchOptions,
+        args: [...launchOptions.args.filter((arg) => arg !== '--disable-gpu'), '--ignore-gpu-blacklist'],
```
Comment (tonypls) [inline web/cypress/plugins/index.js:26]: So we always want to use GPU's for some reason?

**Review by tonypls — APPROVED** [review]

File: web/cypress/plugins/index.js (hunk 1 above)
Comment (Ugzuzg) [reply web/cypress/plugins/index.js:26]: Judging by the error message I was getting in Chromium launched from Cypress, the map is rendered using WebGL, for which the GPU is required.

Comment (madsnedergaard) [reply web/cypress/plugins/index.js:26]: Would be nice to add this explanation as a comment to the code :)

Comment (Ugzuzg) [reply web/cypress/plugins/index.js:28]: Done.

**Review by madsnedergaard — APPROVED** [review]
Nice!

**Pull request merged**